rs/nervous_system/tools/release-runscript/src/main.rs

```diff
@@ -54,14 +60,23 @@ struct CreateForumPost {
 }
 #[derive(Debug, Parser)]
 struct ScheduleVote {
+    #[arg(long)]
+    commit: String,
     #[arg(long, num_args = 0..,)]
     nns_proposal_ids: Vec<String>,
     #[arg(long, num_args = 0..,)]
     sns_proposal_ids: Vec<String>,
 }
 
 #[derive(Debug, Parser)]
-struct UpdateChangelog;
+struct UpdateChangelog {
+    #[arg(long)]
+    commit: String,
```

Comment:
NCR

I think this can (more reliably?) be obtained from the proposal.

Reply:
It can but I don't have those helpers handy since they're written in bash :(